p5/sketch/userspace.py

```diff
@@ -48,6 +48,8 @@
     "save",
     "is_looping",
     "set_frame_rate",
+    # TODO: Add docs for pixel density
```

Comment:
I recommend adding docs along with the functionality in the same PR. It's easier to make sure that docs are in sync with the code. Plus, I don't think writing lots of docs for lots of function in one go is super enjoyable :D

Reply:
Aside from this minor point, this PR looks great!

Reply:
Yes, I agree. I will keep updating the docs as well.